source/Camera/Basic/Capture2D.m

```diff
@@ -0,0 +1,52 @@
+try
+    app = zividApplication; 
+
+    disp('Connecting to camera')
+    camera = connectCamera(app);
+
+    disp('Setting the capture settings');
+    settings2D = Zivid.NET.Settings2D();
+    settings2D.Iris = 22;
+    settings2D.ExposureTime = Zivid.NET.Duration.FromMicroseconds(10000);
+    settings2D.Gain = 1.0;
+
+    disp('Capture a 2D frame');
+    frame2D = camera.Capture2D(settings2D);
+
+    disp('Get RGBA8 image from Frame2D');
+    image = NET.invokeGenericMethod(frame2D,'Image',{'Zivid.NET.RGBA8'});
+
+    disp(['Image Width: ',  num2str(image.Width), ' Height: ',  num2str(image.Height)]);
+    disp(['Pixel (0, 0): R=', num2str(image.Item(1).r), ', G=', num2str(image.Item(1).g), ', B=', num2str(image.Item(1).b), ', A=', num2str(image.Item(1).a)]);
+
+    byteArray = uint8(image.ToArray);
+    disp(['First four bytes:  [0]: ',num2str(byteArray(1)),', [1]: ',num2str(byteArray(2)),', [2]: ',num2str(byteArray(3)),', [3]: ',num2str(byteArray(4))]);
+
+    resultFile = 'image.png';
```

Comment:
Use difference name, as image is a standard name in matlab.

Reply:
Changed to image2D